style.css

```diff
@@ -0,0 +1,89 @@
+body {
+        background-image: url('background.png');
+        background-repeat: no-repeat;
+        background-color: #000;
+        max-width: 2049px;
+}
+
+header {
+          text-align: center;
+          border-bottom-style: solid;
+          border-bottom-color: white;
+          border-bottom-width: 1px
+}
+
+header p {
+            width: 650px; 
+            display: inline-block;
+            font-family: 'Gotham Book', sans-serif;
+            font-size: 16px;
+            color: #fff;
+}
+
+h1 {
+      font-family: 'Gotham', sans-serif;
+      font-size: 18px;
+      font-weight: bold;
+      color: #fff;
+}
+
+h2 {
+      font-family: 'Gotham Book', sans-serif;
+      font-size: 16px;
+      color: #fff;
+      opacity: 0.5;
+}
+
+#section-title {
+                  text-align: center
+}
+
+.team-member img {
+                    border-radius: 50
+}
+
+h3 {
+      font-family: 'Gotham', sans-serif;
+      font-size: 18px;
+      font-weight: bold;
+      color: #fff;
+}
+
+h4 {
+      font-family: 'Gotham Book', sans-serif;
+      font-size: 18px;
+      font-style: italic;
+      color: #fff;
+      opacity: 0.5}
+
+ul {
+      margin: 0;
+      padding: 0;
+      list-style-type: none;
+      text-align: center;
+}
+
+ul li { 
+         display: inline-block;
+         max-width: 650px;
+         margin: 10px
+}
+
+ul p {
+        max-width: 400px;
+        text-align: left;
+        font-family: 'Gotham Book', sans-serif;
+        font-size: 15px;
+        color: #fff;
+}
+
+
+footer {
+          text-align: center;
```

Comment:
Indentation is this file is still weird; let's just indent all of the rules by one tab (two spaces) :)

Reply:
i seem to have 2 spaces for all the rules on my screen, maybe we can look at this over screenshare? :)